frontend/packages/component-library-styles/CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+# Contributing to Component Library Styles
+
+Thank you for considering contributing to the Component Library Styles! 🎉  
+We welcome all contributions — whether it’s a bug fix, new token, documentation improvement, or something else
+entirely.  
+This guide will help you get started quickly and ensure a smooth contribution process. To get the full picture of the
+project, please refer to the [README.md](./README.md) file.
+
+## Table of Contents
+
+- [🛣️ Contribution Roadmap](#-contribution-roadmap)
+- [🎨 How to Add a New Style or Token](#-how-to-add-a-new-style-or-token)
+- [🚦Production-ready Checklist](#production-ready-checklist)
+- [🎯 Coding Standards](#-coding-standards)
+- [🧪 How to Test Your Styles](#-how-to-test-your-styles)
+- [🔀 Commit Message Guidelines](#-commit-message-guidelines)
+- [📚 Need Help?](#-need-help)
+- [💡 Tips for a Smooth PR Review](#-tips-for-a-smooth-pr-review)
+- [✅ How to Get Your PR Approved Faster](#-how-to-get-your-pr-approved-faster)
+
+## 🛣️ Contribution Roadmap
```

Comment:
This is specific to internal contributors, how do external contributors contribute? Can the documentation be made general enough to support both external and internal contributors? For example, could we just create a PR and using [Codeowners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners), we auto assign the `design` group. We can then subscribe `#ask-design-system` to receive messages from Github if assigned to the design group and then anyone in that group can approve it. 

This way the documentation is general and does not rely on a specific person and removes the need for a contributor to have access to Slack.

Reply:
Thanks for spotting this!
I did miss one thing here - and that is that if one doesn't have access to our slack channel - he/she should reach out through github issues and github prs. All the other steps are already available for everyone through github.

It should already be somewhere in the docs, I just forgot to add it here. I'll find this in the docs and will add here